### PR TITLE
CRe margins: hide bottom menu on change

### PR DESCRIPTION
**Pull request by poire-z**
So we can immediately see how's the bottom margin:

<kbd>margins_tap_dismiss</kbd>

(As proposed in https://github.com/koreader/koreader/pull/4945#issuecomment-488571117)

### CODE REVIEW

File: frontend/apps/reader/modules/readertypeset.lua [hunk 1]
```diff
@@ -406,16 +410,19 @@ function ReaderTypeset:onSetPageMargins(margins, silent)
     end
     self.ui.document:setPageMargins(left, top, right, bottom)
     self.ui:handleEvent(Event:new("UpdatePos"))
-    if not silent then
+    if refresh_callback then
         -- Show a toast on set, with the unscaled & scaled values
         UIManager:show(InfoMessage:new{
             text = T(_([[
     Margins set to:
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readertypeset.lua]: @poire-z 
Is this intentionally all indented? It seems like it should be:

```lua
Margins set to

    indented
    indented

Click to dismiss
```

Comment (poire-z) [reply frontend/apps/reader/modules/readertypeset.lua]: That was from @Niluje PR #4945 (to which I added "Tap to dismiss").
(I like it all indented, and I think you once lectured me that if I wanted this visual effect, I'd have to use some left padding - and I went just removing the indentation :)
So, no opinion :)

Comment (NiLuJe) [reply frontend/apps/reader/modules/readertypeset.lua]: I don't *think* I made that decision consciously, FWIW, so, err, no opinion either ;).

Comment (Frenzie) [reply frontend/apps/reader/modules/readertypeset.lua]: In that case I'll get rid of it. It just adds additional complexity to translating the string.

**Pull request merged**